auxiliary.py

```diff
@@ -5,11 +5,11 @@
 
 
 uco = mpatches.Patch(color=(1., 1., 0.), label='UCO')
-buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer')
+buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer layer')
```

Comment:
This is still unclear to me. What material is it made of? Buffer and layer are both geometric descriptions, but the labels should ideally convey what materials (not what shapes)

Reply:
The buffer layer, IPyC, OPyC, compact matrix, and fuel block are all graphite. If I use the same color for all of them, I would not distinguish the different components.

Reply:
I'll go with UCO, porous carbon, IPyC, SiC, OPyC, and Graphite Compact Matrix.